.buildkite/scripts/common.sh

```diff
@@ -30,10 +30,6 @@ XPACK_MODULE_PATTERN="^x-pack\\/[a-z0-9]+beat\\/module\\/([^\\/]+)\\/.*"
 # define if needed run cloud-specific tests for the particular beat
 [ -z "${run_xpack_metricbeat_aws_tests+x}" ] && run_xpack_metricbeat_aws_tests="$(buildkite-agent meta-data get run_xpack_metricbeat_aws_tests --default "false")"
 
-libbeat_changeset=(
-  "^libbeat/.*"
-  )
-
```

Comment:
is this on purpose? given that the PR is about `x-pack/libbeat`

Reply:
libbeat pipeline has already merged.
Also, in this pipeline I've added one part from libbeat (we discussed that earlier): https://github.com/elastic/beats/blob/cd81faa4705b758cf7ca9052d9da5e44fa04f9c4/.buildkite/pipeline.yml#L268